pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -92,6 +93,11 @@ sleep 5
 done`
 )
 
+const (
+	redColor   = "\u001b[31;1m"
```

Comment:
This is specific to terminals that support ANSI color codes, but o.fatal checks `kterm.AllowsColorOutput(o.IOStreams.ErrOut)`; meaning that color will be printed only for terminals that support color output.

The same approach is already used when a "Warning:" message is printed in yellow - in the libraries that `oc must-gather` is using in k8s.io/client-go/rest/warnings.go:
https://github.com/kubernetes/client-go/blob/0a514be22c2b4718e71c09dbd6fc3841f5879432/rest/warnings.go#L124

The Color value is set by NewOcCommand:
https://github.com/openshift/oc/blob/36f5a453c756a2a1da548fb996bdfac98cd1df3d/pkg/cli/cli.go#L174